src/dealWizard/dealWizard.types.ts

```diff
@@ -0,0 +1,90 @@
+import { IWizardConfig } from "services/DealWizardService";
```

Comment:
can we make this service, and all its types, agnostic to deals?  so can apply to any wizard?